op-batcher/rpc/config.go

```diff
@@ -0,0 +1,33 @@
+package rpc
+
+import (
+	opservice "github.com/ethereum-optimism/optimism/op-service"
+	oprpc "github.com/ethereum-optimism/optimism/op-service/rpc"
+	"github.com/urfave/cli"
+)
+
+const (
+	EnableAdminFlagName = "rpc.enable-admin"
+)
+
+func CLIFlags(envPrefix string) []cli.Flag {
+	return []cli.Flag{
+		cli.BoolFlag{
+			Name:   EnableAdminFlagName,
+			Usage:  "Enable the admin API (experimental)",
+			EnvVar: opservice.PrefixEnvVar(envPrefix, "RPC_ENABLE_ADMIN"),
+		},
```

Comment:
this could also be moved to `flags/flags.go` and this file removed

Reply:
This seems in line with how we define other flags / optional sub-services